rest-server/admin/src/main/scala/com/actionml/admin/Administrator.scala

```diff
@@ -40,7 +40,7 @@ abstract class Administrator extends LazyLogging {
     engineJson: Option[String] = None,
     dataDelete: Boolean = false,
     force: Boolean = false,
-    input: Option[String] = None): Validated[ValidateError, Boolean]
+    input: Option[String] = None): Validated[ValidateError, Unit]
```

Comment:
This is a core API we don't change these without agreed reasons because all templates must provide them. Generally the Server is more flexible than the Template APIs which I will be resistant to changing because they have to do with Machine learning workflow. I do change them but only to fit a requirement of our example Template, the Contextual Bandit.

We will eventually have several Templates that are separately compiled and are expected to implement the core Template APIs, if we change anything there all Templates will break and have to be modified at the API level. These will eventually be annotated so we don't change them and Template developers can rely on them.

So why pass no information back on Valid input? We convert sometimes from internal `Validated[ValidateError, SomeCaseClass]` to `Validated[ValidateError, Boolean]` when the case class has no further use but Unit?